### ISSUE TITLE
[NBug] Unable to parse git command output.

### ISSUE THREAD

**Issue by bartkrekelberg** [gitextensions/gitextensions#7317]
:warning: The sections below must be filled in and this text must be removed or the issue will be closed.

## Current behaviour
Opening a repository with a git annex crashes

## Expected behaviour
Not crashing

## Steps to reproduce

Any time I open a git annex repository, gitextensions crashes.

## Error Details
```
System.Exception: Unable to parse git command output.
   at GitCommands.GitModule.<GetRemotesAsync>g__ParseRemotes|153_0(IEnumerable`1 lines)
   at GitCommands.GitModule.<GetRemotesAsync>d__153.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitUI.BranchTreePanel.RepoObjectsTree.RemoteBranchTree.<LoadNodesAsync>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitUI.BranchTreePanel.RepoObjectsTree.Tree.<ReloadNodesAsync>d__27.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitUI.BranchTreePanel.RepoObjectsTree.<>c__DisplayClass69_0.<<AddTree>b__1>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at GitUI.ThreadHelper.<>c__DisplayClass13_0.<<FileAndForget>b__0>d.MoveNext()
```

## Additional information
Trying to open a repository that has a git annex...

## Environment
- Git Extensions 3.2.1.6628
- Build bc07663c3c893f1d896520e0bdcfbe35cfe4b729
- Git 2.23.0.windows.1
- Microsoft Windows NT 10.0.18362.0
- .NET Framework 4.8.4018.0
- DPI 192dpi (200% scaling)

**Comment by pmiossec** [comment]
Hopefully solved by #7214 (See #7182).

You could try the [last built development version](https://github.com/gitextensions/gitextensions#downloads) to see if it's fixed.

**Comment by gerhardol** [comment]
Multiple push url are not supported in GE 3.2
Similar, incorrect remote configuration can cause this problem
